smoke-tests/designer/wdio.conf.js

```diff
@@ -1,6 +1,6 @@
 const { hooks } = require("./support/hooks");
 const drivers = {
-  chrome: { version: "100.0.4896.60" }, // https://chromedriver.chromium.org/
+  chrome: { version: "102.0.5005.61" }, // https://chromedriver.chromium.org/
```

Comment:
necessary?

Reply:
Yes, this is a very annoying recurring issue that cause the pipeline to fail if chromedriver version isnt up to date. We may want an issue ticket raised on the main repo to look at ways to automate the update